docs/topics/about-vscode-extension.adoc

```diff
@@ -1,5 +1,6 @@
 // Module included in the following assemblies:
 //
+// * docs/getting-started-guide/master.adoc
 // * docs/vs-code-extension-guide/master.adoc
```

Comment:
This module is so similar to the "about-intellij" module that you might want to consider using the same module for both guides, with local attributes for the product names. It's your call.